sherpa-onnx/csrc/offline-recognizer-transducer-impl.h

```diff
@@ -72,6 +73,16 @@ class OfflineRecognizerTransducerImpl : public OfflineRecognizerImpl {
     }
   }
 
+  std::unique_ptr<OfflineStream> CreateStream(
+      const std::vector<std::vector<int32_t>> &context_list) const override {
+    // We create context_graph at this level, because we might have default
+    // context_graph(will be added later if needed) that belongs to the whole
+    // model rather than each stream.
+    auto context_graph = std::make_shared<ContextGraph>(config_.context_score);
+    context_graph->Build(context_list);
```

Comment:
Could you pass `context_list` as an argument to the constructor of ContextGraph and make `Build()` a private method?

Reply:
Yet, might be a good idea.